views/pages/single_fixture.js

```diff
@@ -362,167 +250,220 @@ function handlePhysicalData(physical) {
+    <span class="value">${channel.type}</span>
   </section>`;
 
-  if ('color' in channel) {
+  if (channel.color !== null) {
     str += `<section class="channel-color">
       <span class="label">Color</span>
-      <span class="value"><data data-key="channel-color">${channel.color}</data></span>
+      <span class="value">${channel.color}</span>
     </section>`;
   }
 
-  if ('fineChannelAliases' in channel) {
-    const fineChannelPositions = channel.fineChannelAliases.map(
-      fineAlias => getChannelIndexInMode(fineAlias, mode, switchingChannels) + 1
-    ).filter(position => position !== 0); // filter out not used finer channels in the end of the array
-
-    if (fineChannelPositions.length > 0) {
-      str += '<section class="channel-fineChannelAliases">';
-      str += '  <span class="label">Fine channels</span>';
-      str += '  <span class="value"><data data-key="channel-fineChannelAliases">';
-      str += fineChannelPositions.map((position, index) => {
-        const fineAlias = channel.fineChannelAliases[index];
-        return getChannelHeading(fineAlias) + ` (channel ${position})`;
-      }).join(', ');
-      str += '</data></span>';
-      str += '</section>';
+  const finenessInMode = channel.getFinenessInMode(mode);
+  if (finenessInMode > 0) {
+    str += '<section class="channel-fineChannelAliases">';
+    str += '  <span class="label">Fine channels</span>';
+    str += '  <span class="value">';
+
+    for (let i=0; i<finenessInMode; i++) {
+      const heading = getChannelHeading(channel.fineChannelAliases[i]);
+      const position = mode.getChannelIndex(channel.key) + 1;
+      str +=  `${heading} (channel ${position})`;
     }
 
-    divideChannelDmxValues(channel, channel.fineChannelAliases.length - fineChannelPositions.length);
+    str += '</span>';
+    str += '</section>';
   }
 
-  if ('defaultValue' in channel) {
+  if (channel.hasDefaultValue) {
     str += `<section class="channel-defaultValue">
       <span class="label">Default value</span>
-      <span class="value"><data data-key="channel-defaultValue">${channel.defaultValue}</data></span>
+      <span class="value">${channel.defaultValue}</span>
     </section>`;
   }
 
-  if ('highlightValue' in channel) {
+  if (channel.hasHighlightValue) {
     str += `<section class="channel-highlightValue">
       <span class="label">Highlight value</span>
-      <span class="value"><data data-key="channel-highlightValue">${channel.highlightValue}</data></span>
+      <span class="value">${channel.highlightValue}</span>
     </section>`;
   }
 
-  if ('invert' in channel) {
+  if (channel.invert) {
     str += `<section class="channel-invert">
       <span class="label">Invert</span>
-      <span class="value"><data class="checkbox" data-key="channel-invert" data-value="${channel.invert}">${channel.invert}</data></span>
+      <span class="value"><span class="checkbox" title="${channel.invert}">${channel.invert}</span></span>
```

Comment:
Does `title="true"` really make that much sense? Why using `title` instead of `data-*`